ts/src/bingx.ts

```diff
@@ -1903,13 +1903,28 @@ export default class bingx extends Exchange {
         //     }
         //
         if (typeof response === 'string') {
+            // broken api engine : order-ids are too long numbers (i.e. 1742930526912864656)
+            // and JSON.parse can not handle them in JS, so we have to use .parseJson
+            // however, when order has an attached SL/TP, their value types need extra parsing
+            response = this.fixStringifiedJsonMembers (response);
             response = this.parseJson (response);
```

Comment:
@ttodua I don't think this is working
![image](https://github.com/ccxt/ccxt/assets/43336371/ff6cd552-e8ca-47c6-9b1f-aafc9925fd80)

Reply:
oh. can you tell  which command did you run and what response did you get initially?

Reply:
@ttodua I ran something like this:
`n bingx createOrder "LTC/USDT:USDT" market buy 1 undefined '{"takeProfit":{"stopPrice":100},"stopLoss":{"stopPrice": 50}}'`

Reply:
please re-test now, i had to use `replaceAll` instead of `replace`.